### PR TITLE
feat: Create ubuntu dockerfiles

### PR DESCRIPTION
**Pull request by lfservin**
Adapt versions.sh to remove debian-based images and add ubuntu-based images
Adapt dockerfile template to create a non-root user and pull from ubuntu, rather than debian
All Dockerfiles are generated using update.sh

### CODE REVIEW

File: Dockerfile-linux.template [hunk 1]
```diff
@@ -23,6 +23,13 @@ ENV PATH /usr/local/bin:$PATH
 # > At the moment, setting "LANG=C" on a Linux system *fundamentally breaks Python 3*, and that's not OK.
 ENV LANG C.UTF-8
 
+# create a non-root user "wrk" with user id 8877
+{{ if is_alpine then ( -}}
+RUN adduser -g "wrk" wrk
```
Comment (eric-poitras-wrk) [inline Dockerfile-linux.template:28]: The comment about user id 8877 might need to be moved in the conditional block as this does not make sense for Alpine-generated templates.

**Review by eric-poitras-wrk — APPROVED** [review]
This is ok in the context of unfinished work. Here is a couple of observations:
- README.md should be updated for Wrk context.
- The update script does not work with old bash version installed on mac. We should probably use python to generate the templates since we're replacing ruby to python.
- There is way too much image types. I don't think we need 2 versions of alpine and probably no versions of python below 3.10. Those images are long to build and will induce costs in the CI.
- Remove github workflow to be replaced with CircleCI.
- We're missing a tag scheme that reflect our objective of a weekly release.
- With all the points above, I am not sure it's a good thing to fork the OSS repo.

**Review by eric-poitras-wrk — APPROVED** [review]

File: Dockerfile-linux.template (hunk 1 above)
Comment (lfservin) [reply Dockerfile-linux.template:28]: Dirty secret: we can't run 3.10 until we substitute nose for nose2 in our tests (shh)

Comment (lfservin) [reply Dockerfile-linux.template:28]: it makes sense for every image. every image gets the relevant user add (adduser vs useradd) with a matching user. 
No image should be created without a non-root user

**Comment by lfservin** [comment]
> This is ok in the context of unfinished work. Here is a couple of observations:
> 
>     * README.md should be updated for Wrk context.
Sure, will do it in another branch
> 
>     * The update script does not work with old bash version installed on mac. We should probably use python to generate the templates since we're replacing ruby to python.
Homebrew is always an option
> 
>     * There is way too much image types. I don't think we need 2 versions of alpine and probably no versions of python below 3.10. Those images are long to build and will induce costs in the CI.
right now we're stuck with 9.8 and 9.9 in most cases. 9.10 doesn't work with our current test framework (nose), we need to replace it with nose2.
> 
>     * Remove github workflow to be replaced with CircleCI.
Yes, working on that in the circleCi branch
> 
>     * We need a tag scheme that reflect our objective of a weekly release. We also need to make sure this scheme is supported by renovate.
To be done in the circleci branch
> 
>     * With all the points above, I am not sure it's a good thing to fork the OSS repo.
Well, it's a place to start. We can then improve or pivot away from it. We might need to look into a templating engine in python or whatever we want to use. jsonnet has no future either, so we can't use that

**Review by eric-poitras-wrk — APPROVED** [review]

**Review by eric-poitras-wrk — APPROVED** [review]

**Review by eric-poitras-wrk — APPROVED** [review]

**Pull request merged**